src/main/scala/scalismo/ui/rendering/actor/mixin/ActorScalarGridRange.scala

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This does not seem to be needed. Can we remove it?

Reply:
Yes. I'll remove it and push.